corehq/apps/aggregate_ucrs/migrations/0002_auto_20180824_1319.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-08-24 13:19
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('aggregate_ucrs', '0001_initial_squashed_0008_auto_20180625_1105'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='secondarycolumn',
+            name='aggregation_type',
+            field=models.CharField(choices=[('sum', 'Sum'), ('min', 'Min'), ('max', 'Max'), ('avg', 'Average'), ('count', 'Count'), ('count_unique', 'Count Unique Values'), ('nonzero_sum', 'Nonzero sum? (1 if sum is nonzero else 0)')], max_length=12),
+        ),
+        migrations.AlterField(
+            model_name='secondarytabledefinition',
+            name='join_column_primary',
+            field=models.CharField(max_length=63),
+        ),
```

Comment:
Where did this second alter field on `join_column_primary` come from?

Reply:
good question. it looks like the previous squashed migration [specified a default](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/aggregate_ucrs/migrations/0001_initial_squashed_0008_auto_20180625_1105.py#L59) that was removed and no migration was added for it.